libraries/matrix/impl/src/main/kotlin/io/element/android/libraries/matrix/impl/timeline/MatrixTimelineItemMapper.kt

```diff
@@ -30,6 +37,12 @@ class MatrixTimelineItemMapper(
         val asEvent = it.asEvent()
         if (asEvent != null) {
             val eventTimelineItem = eventTimelineItemMapper.map(asEvent)
+
+
+            if (eventTimelineItem.hasNotLoadedInReplyTo() && eventTimelineItem.eventId != null) {
```

Comment:
Maybe it's not the best point to do this? It's where iOS fetches this data, but maybe we can do this in `MatrixClient` instead.